src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static class Node<T> {
+        private T element;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(T element) {
+            this.element = element;
+        }
+    }
```

Comment:
privat class should be located at the end of the MyLinkedList class
**move after public methods** Nastya commented